starcheck/utils.py

```diff
@@ -64,6 +65,11 @@ def starcheck_version():
     return version
 
 
+@print_traceback_on_exception
+def get_chandra_models_version():
+    return proseco_char.chandra_models_version
```

Comment:
It seemed cheap to just get the version again from proseco since it is already set in that scope.  If we want something to set and fetch a new value in calc_ccd_temps.py that is also obviously an option.

Reply:
I need to fix this up because it is potentially quite confusing, but that `chandra_models_version` module global only gets set the first time a model is actually read:
```
In [1]: from proseco import characteristics as proseco_char                     
In [2]: print(proseco_char.chandra_models_version)                              
None
```
Most likely your code is getting run after the `aca` model was fetched and then you get the expected value.

SO... if the code seems to be giving a reasonable answer now, then leave it. I'm going to look into a better way, probably making `chandra_models_version` also be dynamically determined.

Reply:
Right.  I put the call to get_chandra_models_version down in the printing because basically everything has already happened/been called by that point in the code.  So yes, not awesome but worked in this case.